app/helpers/events_helper.rb

```diff
@@ -16,4 +16,27 @@ def users_for_event(state)
     req = req.where(state: state) if state != 'all'
     user_email = req.map { |e| e.user.email }.uniq
   end
+
+  def check_states(requests)
+    if requests.map { |r| r.event = @event }.count > 1
```

Comment:
I would prefer to do this in sql, like joining the events and then limit the result to 2.